demisto_sdk/commands/validate/validators/PB_validators/PB108_is_valid_task_id.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.tools import is_string_uuid
+from demisto_sdk.commands.content_graph.objects.base_playbook import TaskConfig
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsValidTaskIdValidator(BaseValidator[ContentTypes]):
+    error_code = "PB108"
+    description = "Validate that the task ID and the 'id' under the 'task' field are from UUID format."
+    rationale = "Validate that the task ID and the 'id' under the 'task' field are from UUID format."
+    error_message = "This playbook has tasks with invalid 'taskid' or invalid 'id' under the 'task' field."
+    related_field = "taskid"
+    is_auto_fixable = False
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        """Check whether all playbook tasks has valid taskid and the 'id' under the 'task' field is valid as well.
+        Args:
+            - content_items (Iterable[ContentTypes]): The content items to check.
+        Return:
+            - List[ValidationResult]. List of ValidationResults objects.
+        """
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            invalid_tasks = self.invalid_tasks(content_item.tasks)
+            tasks_error_message = self.error_message
+            for task_id in invalid_tasks:
+                tasks_error_message = (
+                    f"{tasks_error_message}\nTask {task_id} has invalid UUIDs in the fields "
+                    f"{invalid_tasks[task_id]}."
+                )
+            if invalid_tasks:
+                results.append(
+                    ValidationResult(
+                        validator=self,
+                        message=tasks_error_message,
```

Comment:
why not message=self.error_message.format(task_error_message)
where self.error_message = "This playbook has tasks with invalid 'taskid' or invalid 'id' under the 'task' field:\n{0}"?